docs/oidc_testing/README.md

```diff
@@ -0,0 +1,81 @@
+# Bring up a cluster w/ the OIDC code enabled.
+1. Set up dev environment (which you've probably already done)
```

Comment:
"which you've probably already done" can simply be removed.

Reply:
Fixed in cbed0d6